lib/runner/extensions/request.command.js

```diff
@@ -96,9 +100,11 @@ module.exports = {
              * @property {VariableScope} globals
              * @property {Object} data
              * @property {Boolean} replay - Indicates whether this request a replayed request
+             * @property {Number} replayCount - Number of times request was replayed
              */
             context = createContext(payload);
             replayed = context.replay;
```

Comment:
Since you're storing the count now, can you get rid of this flag? (`replayed` is true if count is greater than zero)